plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMigrateProcedure.java

```diff
@@ -136,6 +173,79 @@ public void testMigrateTableWithSmallintType(IcebergFileFormat fileFormat)
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @ParameterizedTest
+    @MethodSource("fileFormats")
+    public void testMigrateTableSchemaEvolution(IcebergFileFormat fileFormat)
+            throws Exception
+    {
+        String randomNameSuffix = randomNameSuffix();
+        String tableNameOneColumn = "test_migrate_one_column_" + randomNameSuffix;
+        String tableNameTwoColumns = "test_migrate_two_columns_" + randomNameSuffix;
+        String hiveTableNameOneColumn = "hive.tpch." + tableNameOneColumn;
+        String hiveTableNameTwoColumns = "hive.tpch." + tableNameTwoColumns;
+        String icebergTableNameTwoColumns = "iceberg.tpch." + tableNameTwoColumns;
+
+        assertUpdate("CREATE TABLE " + hiveTableNameOneColumn + " WITH (format='" + fileFormat + "') AS SELECT 1 col1", 1);
+        assertUpdate("CREATE TABLE " + hiveTableNameTwoColumns + " WITH (format='" + fileFormat + "') AS SELECT 2 col1, CAST(row(10, 20) AS row(x integer, y integer)) AS nested", 1);
+
+        // Copy the parquet file containing only one column to the table with two columns
```

Comment:
This test isn't specific to Parquet. 
```suggestion
        // Copy the file containing only one column to the table with two columns
```